Helpers.js

```diff
@@ -386,4 +386,9 @@
         return total;
     });
 
+    // replace spaces with an HTML ID friendly character
```

Comment:
I think we might be able to remove this file entirely. I don't see anything referencing it. The Gruntfile references `helpers.js` and removing the file locally doesn't seem to break anything 👍